pursuit.cabal

```diff
@@ -85,7 +85,7 @@ library
                  , directory                     >= 1.1        && < 1.3
                  , warp                          >= 3.0        && < 4
                  , data-default
-                 , aeson                         >= 0.6        && < 0.10
+                 , aeson                         >= 0.6        && < 0.12
```

Comment:
The `ToJSON` and `FromJSON` instances that Aeson provides for common data types might have changed between versions 0.x and 0.(x+1), and the parsing of the JSON files that people upload makes use of those instances. We need to be using the same version as what the compiler is using, essentially. Really the bounds should have been `>= 0.9 && < 0.10` before, I think.